src/components/views/SignupPage.jsx

```diff
@@ -10,6 +10,7 @@ import queryString from 'query-string';
 const SignupPage = (props) => {
   const isAuth = isLoggedIn();
   const dispatch = useDispatch()
+  
```

Comment:
I think we can use the `useLocation` hook from `react-router-dom` and not pass it as props

Reply:
NIce! Done